lib/reports/ppl-applications/index.js

```diff
@@ -70,6 +70,10 @@ module.exports = ({ db, query: params, flow }) => {
       .where({ 'projects.id': record.data.id })
       .first()
       .then(project => {
+        if (moment(project.issue_date).isBefore('2019-08-01')) {
+          // ignore PPLs which have had their issue date changed to pre-aspel
+          return [];
+        }
```

Comment:
how come this isn't just a where clause on the query?

Reply:
Because the query is querying tasks, not projects.

Reply:
And before you suggest `modelData` that stores the value at the point of application, so won't catch these cases.